opentelemetry-instrumentation/src/opentelemetry/instrumentation/auto_instrumentation/sitecustomize.py

```diff
@@ -26,12 +27,26 @@
 
 
 def auto_instrument():
-    for entry_point in iter_entry_points("opentelemetry_instrumentor"):
+    package_to_exclude = Configuration().get("DISABLED_INSTRUMENTATIONS", None)
+    if package_to_exclude:
+        package_to_exclude = package_to_exclude.split(",")
+        packages_to_instrument = [
+            entry_point
+            for entry_point in iter_entry_points("opentelemetry_instrumentor")
+            if entry_point.name not in package_to_exclude
+        ]
+    else:
+        packages_to_instrument = (
+            entry_point
+            for entry_point in iter_entry_points("opentelemetry_instrumentor")
+        )
+
+    for package in packages_to_instrument:
         try:
-            entry_point.load()().instrument()  # type: ignore
-            logger.debug("Instrumented %s", entry_point.name)
+            package.load()().instrument()  # type: ignore
```

Comment:
Sorry, this comment was lost yesterday when I was reviewing. I think `entry_point` is a more accurate name:

``` python
In [1]: from pkg_resources import iter_entry_points

In [2]: for entry_point in iter_entry_points("opentelemetry_instrumentor"):
   ...:     print(type(entry_point))
   ...: 
<class 'pkg_resources.EntryPoint'>
```

Reply:
Also, I think we could get rid of the filter above and have an if condition inside this loop. That'd allow us to log a statement indicating we are ignoring a specific instrumentation.